spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/reload/ConfigReloadAutoConfigurationTest.java

```diff
@@ -34,16 +39,20 @@
  * To test if either kubernetes, kubernetes.configmap, or reload is disabled, then the
  * detector and update strategy beans won't be available.
  **/
-
+@EnableKubernetesMockClient(crud = true, https = false)
 public class ConfigReloadAutoConfigurationTest extends KubernetesConfigTestBase {
 
 	private static final String APPLICATION_NAME = "application";
 
-	@BeforeClass
-	public static void setUpBeforeClass() {
+	// injected because of @EnableKubernetesMockClient. Because of the way
```

Comment:
I hope this comment makes sense of why the configuration has moved in here from the common class. It is just the way `@EnableKubernetesMockClient` "injects" fields.